roles/template-configs/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Template Browbeat configuration
+  template:
+    "src={{ browbeat_config_file }} \
```

Comment:
Can we use the more recent 
template:
   src:
   dst:

Syntax

Reply:
resolved in the updated PS